pkg/builder/builder.go

```diff
@@ -681,11 +726,7 @@ func makeStartupProbe(vdb *vapi.VerticaDB) *corev1.Probe {
 
 // makeLivenessProbe will return the Probe object to use for the liveness probe.
 func makeLivenessProbe(vdb *vapi.VerticaDB) *corev1.Probe {
-	// We check if the TCP client port is open. We used this approach,
-	// rather than issuing 'select 1' like readinessProbe because we need
-	// to minimize variability. If the livenessProbe fails, the pod is
-	// rescheduled. So, it isn't as forgiving as the readinessProbe.
-	probe := makeVerticaClientPortProbe()
+	probe := makeDefaultLivenessProbe(vdb)
```

Comment:
Will we be able to use the overrides to change the probe back to checking the TCP client port? Similar question to change back to canary query.

Reply:
Yes we will. I will add a unit test.

Reply:
> Yes we will. I will add a unit test.

But I guess we will need to mount the superuser password in all cases.

Reply:
No you shouldn't have to. The builder should be smart enough to only mount the superuser password if it is required in the probe (see `probeContainsSuperuserPassword`)